ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlans.j2

```diff
@@ -4,10 +4,10 @@
 !
 vlan {{ vlan }}
    name {{ vlans[vlan].name }}
-{%         if vlans[vlan].state is defined and vlans[vlan].state == "suspend" %}
+{%         if vlans[vlan].state is arista.avd.defined("suspend") %}
    state {{ vlans[vlan].state }}
 {%         endif %}
-{%         if vlans[vlan].trunk_groups is defined %}
+{%         if vlans[vlan].trunk_groups is arista.avd.defined %}
```

Comment:
This check is now part of `natural_sort` so you don't need it before the for-loop